FAQ.md

```diff
@@ -16,7 +16,11 @@ WARNING: Please consider reporting this to the maintainers of org.apache.ibatis.
 WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
 WARNING: All illegal access operations will be denied in a future release
 ```
-This warning is shown when using JDK 9 and later. It will need to be fixed by MyBatis.
+This warning is shown when using JDK 9 and later.<br>
+Mybatis hat this issue already fixed in version 3.5.3. see https://github.com/mybatis/mybatis-3/issues/1156 <br>
```

Comment:
Some small findings:

* MyBatis / mybatis -> MyBatis
* hat -> has
* `<br>` -> "  " (twice)
* camunda -> Camunda (twice)

Reply:
Oh wow :facepalm: This happens when you rush a little... done